pkg/dashboard/static/index.html

```diff
@@ -110,20 +110,17 @@ <h2 class="mb-3">name-of-repo</h2>
             <div class="mb-5">
                 <span class="rounded rounded-1 me-2 p-1 px-2 bg-tag text-dark">URL: <span class="url fw-bold">http://somerepo/somepath</span></span>
             </div>
-
-            <!-- TODO
-            <div class="float-end">
-                   <input class="form-control form-control-sm" type="text" placeholder="Filter...">
+            <div class="row py-2 mb-3">
```

Comment:
The visual of this control in TODO was chosen to be less agressive and take less space on screen. I would suggest to stick to it for consistency.

Reply:
that disturbs the entire layout 

![image](https://user-images.githubusercontent.com/17697790/199016431-582469ea-b247-47ec-b2c3-4b381b77e793.png)